include/deal.II/fe/block_mask.h

```diff
@@ -80,6 +80,14 @@ class BlockMask
    */
   BlockMask() = default;
 
+  /**
+   * Deprecated constructor allowing implicit conversion.
+   */
+  template <typename = void>
+  DEAL_II_DEPRECATED_EARLY_WITH_COMMENT(
+    "Implicit conversions from std::vector<bool> to BlockMask are deprecated!")
+  BlockMask(const std::vector<bool> &block_mask);
```

Comment:
How does this work? Does the compiler select this one for implicit conversions because the other compiler is explicit?

Reply:
Yes.